autorest/codegen/templates/operations_container_init.py.jinja2

```diff
@@ -7,7 +7,7 @@ from .{{ ("_operations" if code_model.options['combine_operation_files'] else op
 # coding=utf-8
 {{ code_model.options['license_header'] }}
 
-{% if not code_model.options["add_python_3_operation_files"] %}
+{% if (not code_model.options["add_python_3_operation_files"]) or async_mode %}
```

Comment:
in the case of aio, we only have to generate one set of operations files, because we don't need a typed and untyped version. In this case, I think it's better to leave the name as `_operations.py`, because `_operations_py3.py` is a grosser name, and we don't need to give it a grosser name if we don't need to. So I really think we should leave the code as is here.